frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -369,14 +389,19 @@ export const PublicFormProvider = ({
     data?.form.responseMode === FormResponseMode.Encrypt &&
     data.form.payments_field.enabled
 
+  const hasMyInfoError = !!(
+    data?.errorCodes &&
+    data.errorCodes.find((errorCode) => errorCode === ErrorCode.myInfo)
```

Comment:
/nit `data?.errorCodes.find((errorCode) => errorCode === ErrorCode.myInfo)`

Reply:
@KenLSM , i have simplified it to: 

`const hasMyInfoError = !!data?.errorCodes?.find(
    (errorCode) => errorCode === ErrorCode.myInfo,
  )`